pkg/controller/storagecluster/stork.go

```diff
@@ -821,15 +832,21 @@ func (c *Controller) createStorkSchedDeployment(
 		}
 	}
 
+	nodeTopologyLabelKeys, err := k8sutil.GetNodeTopologyLabelKeys()
```

Comment:
We could call the `util.GetTopologySpreadConstraints` here and then pass that to `util.HasTopologySpreadConstraintsChanged` to compare the entire object. Wdyt?